lib/libutee/trace_ext.c

```diff
@@ -108,4 +117,9 @@ int puts(const char *str __unused)
 	return 0;
 }
 
+int putchar(int c __unused)
+{
+	return c;
```

Comment:
minor: remove `__unused` above.
minor: `return (int)(unsigned char)c;` (char is expected to be cast as an unsigned char)?